src/CaptureService/include/CaptureService/StartStopCaptureRequestWaiterImpl.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_SERVICE_START_STOP_CAPTURE_REQUEST_WAITER_IMPL_H_
+#define CAPTURE_SERVICE_START_STOP_CAPTURE_REQUEST_WAITER_IMPL_H_
+
+#include <grpcpp/grpcpp.h>
+
+#include "CaptureService/StartStopCaptureRequestWaiter.h"
+#include "GrpcProtos/services.grpc.pb.h"
+#include "GrpcProtos/services.pb.h"
+
+namespace orbit_capture_service {
+
+class StartStopCaptureRequestWaiterImpl : public StartStopCaptureRequestWaiter {
```

Comment:
Again, I would rather add the name "Grpc" to the class name.

Reply:
Done! Thanks!